flowmachine/Dockerfile

```diff
@@ -8,10 +8,10 @@ WORKDIR /app
 COPY Pipfile* /app/
 RUN pip install pipenv && pipenv install --deploy
 COPY . /app/
-RUN pipenv install .
+RUN pipenv run pip install .
```

Comment:
Why do you prefer to install flowmachine via pip run through pipenv, instead of just plain pipenv?

Reply:
Just to record the rationale here - we should not be relocking in the dockerfile, because the build won't be what we've just tested, and it bloats the resulting image massively. Skipping the lock is fine, because we've explicitly installed flow machine's dependencies.